README.md

```diff
@@ -34,6 +42,14 @@ Only two things basically:
 
 [A live demo site](https://geobrain.csiss.gmu.edu/Geoweaver) is available.
 
+
+https://github.com/JYOSHREDDY/Geoweaver/assets/102822880/15bad419-0449-4301-9a56-b49c6106ea82
+
+![Screen Recording 2024-03-16 at 11 19 55 PM](https://github.com/JYOSHREDDY/Geoweaver/assets/102822880/8d6fb615-1430-4fe5-9728-e99586c9fd2a)
```

Comment:
The processes in the video are failing. Please use a different worklfow, the SWE workflow has a few dependencies / assets on an external server which require you to download a few gigs of files.

Also, the recording source should be ESIPFed repo, as of now it is coming from your github account. Please verify